code/game/objects/items/devices/doorCharge.dm

```diff
@@ -27,7 +27,8 @@
 
 /obj/item/device/doorCharge/examine(mob/user)
 	..()
-	if(user.mind in ticker.mode.traitors) //No nuke ops because the device is excluded from nuclear
+	//if(user.mind in ticker.mode.traitors) //No nuke ops because the device is excluded from nuclear
+	if(FALSE) // TODO IF A TRAITOR LOOKS AT IT LOL
```

Comment:
Having an item that has a difference examine depending on antag status is fine, but this is the WEIRDEST FUCKING CHECK that includes, for example, revenants. Also doesn't include observers.